lib/Core/Executor.cpp

```diff
@@ -290,10 +290,17 @@ namespace {
 		    clEnumValEnd),
 		  cl::ZeroOrMore);
 
+#if LLVM_VERSION_CODE < LLVM_VERSION(3, 0)
   cl::opt<unsigned int>
   StopAfterNInstructions("stop-after-n-instructions",
                          cl::desc("Stop execution after specified number of instructions (default=0 (off))"),
                          cl::init(0));
+#else
+  cl::opt<unsigned long long>
```

Comment:
`unsigned long long` is already accepted by the LLVM Parser class for Options. I am not sure if `uint64_t` is. I will check it.